testing/py-ostinato/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-ostinato
+_pkgname=python-ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="A python agent for the ostinato controller"
+url="http://ostinato.org/"
+arch="noarch"
+license="GPLv3"
+depends="python"
```

Comment:
oh ok. I thought for python packages we didn't need to specify the dependencies, sorry.